platformio.ini

```diff
@@ -1,49 +1,20 @@
 ; PlatformIO Project Configuration File
 ;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
 
-;-------------------------------------------------------
-; global PIO parameters                                |
-;-------------------------------------------------------
-[platformio]
-;default build enfironment for PIO project
-default_envs = esp32-s3-devkitc-1
-
-;Path to source files relative to PIO project
-; see https://docs.platformio.org/en/stable/projectconf/sections/platformio/options/directory/src_dir.html
-src_dir = src
-
-;switch "include" path to be src as well, as we decide to have headers and sources in same directories
-; this prevents the build environment from complaining the path not existing
-include_dir = src
-
-;-------------------------------------------------------
-; common parameters for all environments [env:...]     |
-;-------------------------------------------------------
-[env]
-;filter for source file directory when building
-; see https://docs.platformio.org/en/stable/projectconf/sections/env/options/build/build_src_filter.html
-build_src_filter = +<*> -<.git/> -<.svn/>
-
-;general dependencies
-; for version requirements see https://docs.platformio.org/en/stable/core/userguide/pkg/cmd_install.html#cmd-pkg-install-requirements
-lib_deps = 
-  neroroxxx/RoxMux@^1.6.2
-  adafruit/Adafruit SSD1306@~2.5.3
-  simsso/ShiftRegister74HC595@^1.3.1
-
-;general serial monitor baud rate
-monitor_speed = 115200
-
-;-------------------------------------------------------
-; ESP32 build environment                              |
-;-------------------------------------------------------
 [env:esp32-s3-devkitc-1]
 platform = espressif32
 board = esp32-s3-devkitc-1
 framework = arduino
-;;add dependencies to general dependencies from [env]
```

Comment:
this is an example of how the config of [env] can be expanded. If "${env.lib_deps}" was not used, the parameter would be overwritten.